lldb/test/API/lang/swift/cxx_interop/forward/test-cxx-class/Makefile

```diff
@@ -0,0 +1,5 @@
+SWIFT_SOURCES := main.swift
+
+SWIFTFLAGS_EXTRAS = -Xcc -I$(SRCDIR) -Xfrontend -enable-experimental-cxx-interop -Xfrontend -validate-tbd-against-ir=none 
```

Comment:
Should this be a SWIFT_CXX_INTEROP := 1 that works like SWIFT_OBJC_INTEROP?https://github.com/apple/llvm-project/blob/a3a1b62aa5f606a7afae17db5a528e566474bdba/lldb/packages/Python/lldbsuite/test/make/Makefile.rules#L562

Reply:
Sounds like a good idea to me!